src/fu-util.c

```diff
@@ -303,6 +303,7 @@ fu_util_prompt_for_device (FuUtilPrivate *priv, GError **error)
 
 	/* TRANSLATORS: get interactive prompt */
 	g_print ("%s\n", _("Choose a device:"));
+	g_print ("0.\t%s\n", _("Cancel"));
```

Comment:
You need a translator comment here, something like `TRANSLATORS: this is to abort the interactive prompt`